File: programming_fundamentals/07.functions/02.exercise/04.odd_and_even_sum.py
def sum_even_odd(num:int):
    sum_even = 0
    sum_odd = 0
    num = str(num)
    for char in num:
        char = int(char)
        if char%2==0:sum_even+=char
        else: sum_odd+=char
    return f'Odd sum = {sum_odd}, Even sum = {sum_even}'

print(sum_even_odd(int(input())))

File: 06.js_front-end/03.functionsAndStatements/02.exercise/04.oddAndEvenSum.js
solve = (num) => console.log(`Odd sum = ${num.toString().split('').map(x=>Number.parseInt(x))
.reduce((sum, num) => sum += num%2==1?num:0, 0)}, Even sum = ${num.toString().split('')
.map(x=>Number.parseInt(x)).reduce((sum, num) => sum += num%2==0?num:0, 0)}`)

solve(1000435);
solve(3495892137259234);